From 4cb97982f88d8fb623ace5a9511198e442e993ba Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 20 Oct 2023 17:15:25 +0100
Subject: [PATCH] dts: bcm2712: Add the krnbt parameter

Add a Pi 5 implementation of the krnbt parameter, for symmetry and
for tinkering purposes.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2712-rpi-5-b.dts | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/bcm2712-rpi-5-b.dts
+++ b/arch/arm/boot/dts/bcm2712-rpi-5-b.dts
@@ -809,6 +809,7 @@ spi10_cs_pins: &spi10_cs_gpio1 {};
 		i2c_baudrate = <&i2c_arm>, "clock-frequency:0";
 		i2c_arm_baudrate = <&i2c_arm>, "clock-frequency:0";
 		i2c_vc_baudrate = <&i2c_vc>, "clock-frequency:0";
+		krnbt = <&bluetooth>, "status";
 		nvme = <&pciex1>, "status";
 		pciex1 = <&pciex1>, "status";
 		pciex1_gen = <&pciex1> , "max-link-speed:0";
